### PR TITLE
PLAT-1730: Update GA measures with the 2 hours span.

### PR DESCRIPTION
**Pull request by CRMGB**
Added the 2 hours not counting span to all GA measures and another row to the tables to make sense of it.

### CODE REVIEW

**Comment by fradeve** [comment]
LGTM :+1:

File: docs/up-ga/downloads/v1.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 <div id="value-display"></div>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-  Total downloads, as determined by Google Analytics. To count multiple downloads of the same file by the same user, there must be a gap of at least 30 minutes from the last download, even if that previous download was not counted.
+  Total downloads, as determined by Google Analytics. To count multiple downloads of the same file by the same user, there must be a gap of at least 30 minutes from the last download, even if that previous download was not counted. NOTE: A view that occurs 2 hours after the first download will not be counted. See the last entry in the table below; it should be counted because it is 30 minutes after the last download, but because it is 2 hours after the first download, it is not counted.
```
Comment (rowan08) [inline docs/up-ga/downloads/v1.md:14]: Please update the definition to the following:

Total downloads, as determined by Google Analytics. To count multiple downloads of the same file by the same user, there must be a period of at least 30 minutes of inactivity on the website, by the user.

Note: The user's activity is logged in a session. The example below considers one type of interaction on a single page; however, a single session can contain interactions involving page views, downloads, and reads (even across different pages on the website). These interactions are logged within the same session if they occur before the session timeout. The user's session must end before another download of a specific file can be counted for that user (i.e. as part of an different session).

File: docs/up-ga/downloads/v1.md [hunk 2]
```diff
@@ -60,6 +60,11 @@ For Example:
         <td>34</td>
         <td>1</td>
     </tr>
+    <tr>
+        <td>5:10 pm</td>
+        <td>31</td>
+        <td>0</td>
+    </tr>
```
Comment (rowan08) [inline docs/up-ga/downloads/v1.md:67]: I think this can be removed

File: docs/up-ga/reads/v1.md [hunk 3]
```diff
@@ -11,7 +11,7 @@
 <div id="value-display"></div>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-  Total views on pages where the work can be read, as determined by Google Analytics. To count multiple reads on the same page by the same user, there must be a gap of at least 30 minutes from the last read of that page, even if that previous read was not counted.
+  Total views on pages where the work can be read, as determined by Google Analytics. To count multiple reads on the same page by the same user, there must be a gap of at least 30 minutes from the last read of that page, even if that previous read was not counted. NOTE: A read that occurs 2 hours after the first read will not be counted. See the last entry in the table below; it should be counted because it is 30 minutes after the last read, but because it is 2 hours after the first read, it is not counted.
```
Comment (rowan08) [inline docs/up-ga/reads/v1.md:14]: Total views on pages where the work can be read, as determined by Google Analytics. To count multiple reads on the same page by the same user, there must be a period of at least 30 minutes of inactivity on the website, by the user.

Note: The user's activity is logged in a session. The example below considers one type of interaction on a single page; however, a single session can contain interactions involving page views, downloads, and reads (even across different pages on the website). These interactions are logged within the same session if they occur before the session timeout. The user's session must end before another read of a specific page can be counted for that user (i.e. as part of an different session).

File: docs/up-ga/reads/v1.md [hunk 4]
```diff
@@ -61,6 +61,11 @@ For Example:
         <td>34</td>
         <td>1</td>
     </tr>
+    <tr>
+        <td>5:10 pm</td>
+        <td>31</td>
+        <td>0</td>
+    </tr>
```
Comment (rowan08) [inline docs/up-ga/reads/v1.md:68]: Again, I think this can be removed

File: docs/up-ga/sessions/v1.md [hunk 5]
```diff
@@ -12,7 +12,7 @@
 <br>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-Total views of the landing page of a work, as determined by Google Analytics. To count multiple views of the same landing page by the same user, there must be a gap of at least 30 minutes from the last view of that landing page, even if that previous view was not counted.
+Total views of the landing page of a work, as determined by Google Analytics. To count multiple views of the same landing page by the same user, there must be a gap of at least 30 minutes from the last view of that landing page, even if that previous view was not counted. NOTE: A view that occurs 2 hours after the first session will not be counted. See the last entry in the table below; it should be counted because it is 30 minutes after the last session, but because it is 2 hours after the first session, it is not counted.
```
Comment (rowan08) [inline docs/up-ga/sessions/v1.md:15]: Total views of the landing page of a work, as determined by Google Analytics. To count multiple views of the same page by the same user, there must be a period of at least 30 minutes of inactivity on the website, by the user.

Note: The user's activity is logged in a session. The example below considers one type of interaction on a single page; however, a single session can contain interactions involving page views, downloads, and reads (even across different pages on the website). These interactions are logged within the same session if they occur before the session timeout. The user's session must end before another view of a specific page can be counted for that user (i.e. as part of an different session).

File: docs/up-ga/sessions/v1.md [hunk 6]
```diff
@@ -61,6 +61,11 @@ For Example:
         <td>34</td>
         <td>1</td>
     </tr>
+    <tr>
+        <td>5:10 pm</td>
+        <td>31</td>
+        <td>0</td>
+    </tr>
```
Comment (rowan08) [inline docs/up-ga/sessions/v1.md:68]: Again, please remove

File: docs/up-logs/downloads/v1.md [hunk 7]
```diff
@@ -60,10 +60,15 @@ For Example:
             <td>34</td>
             <td>1</td>
         </tr>
+        <tr>
```
Comment (rowan08) [inline docs/up-logs/downloads/v1.md:63]: I'm not sure if you want to remove the up-logs additions? I think they are still accurate so you probably don't need to :thinking:

**Review by rowan08 — CHANGES_REQUESTED** [review]
I've updated the GA definitions. Please replace the definitions where requested and remove the additions to the tables in the ga measures.

File: docs/up-logs/downloads/v1.md (hunk 7 above)
Comment (CRMGB) [reply docs/up-logs/downloads/v1.md:63]: I'll delete the extra row as the up-ga ones for consistency

**Review by rowan08 — APPROVED** [review]
I think it's ok now :+1:

**Pull request merged**